### ISSUE TITLE
Warning in install.packages :   package ‘gmkin’ is not available for this version of R

### ISSUE THREAD

**Issue by Marius-cant-code** [jranke/gmkin#6]
Hi - I am not very experienced in R so I am not entirely sure this is a bug, but I get this error when I follow the indicated steps in the gmkin page (https://pkgdown.jrwb.de/gmkin/):

"Warning in install.packages :
  package ‘gmkin’ is not available for this version of R"

**Comment by jranke** [comment]
Hi, thanks for your interest. I have not updated the Windows binary version of the package in a while. Could you try

```
install.packages("gmkin", type = "source")
```

instead after adding my repository?

**Comment by Marius-cant-code** [comment]
Thanks for getting back to me. Unfortunately, it did not work, but thanks for the suggestion :-)

**Comment by jranke** [comment]
Could you post the error message you got?

**Comment by Marius-cant-code** [comment]
Hi - Sorry for the delayd response. 
I got this error message: 

> install.packages("gmkin", type = "source")

Warning in install.packages :
  unable to access index for repository https://jranke.github.io/drat/src/contrib:
  cannot open URL 'https://jranke.github.io/drat/src/contrib/PACKAGES'
Warning in install.packages :
  unable to access index for repository https://jranke.github.io/drat/src/contrib:
  cannot open URL 'https://jranke.github.io/drat/src/contrib/PACKAGES'
Warning in install.packages :
  package ‘gmkin’ is not available for this version of R
A version of this package for your version of R might be available elsewhere,
see the ideas at
https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages

**Comment by jranke** [comment]
Hi, it seems your R installation cannot access the repository files on github, like

https://jranke.github.io/drat/src/contrib/PACKAGES

I have no idea why this would be the case, these files should be readable for all the world wide web.

**Comment by jranke** [comment]
I just did a fresh installation of the latest R version (4.3.2) on Windows 10, and both installing according to the instructions with 


```
drat::addRepo("jranke")
install.packages("gmkin")
```

and using `, type = "source")` in the install command worked.

Could you also post the error message for your first attempt at the installation?

Also, please post the output of

```
sessionInfo()
```

Kind regards, Johannes

**Comment by Marius-cant-code** [comment]
Hello again - and I just wanted to first say I really appreciate you taking the time help me with this :-) 

I did an install on my home computer now, because I came to suspect that it may have been a problem with security at my work computer. Now there was no problem with the installation. I will have to bring this to my IT department and they should be able to help me I think. 

Thanks again for your help and have great weekend :-)

Best regards
Marius
